### DISCUSSION TITLE
Axis rotation without IPF rotation

### DISCUSSION THREAD

**Discussion by Fjvs1** [mtex-toolbox/mtex#1313] [Ask Anything]
Hi there,

I'm using MTEX 5.7.0, and I'm facing some trouble with the "import wizard" and data rotation ('`rotation.byEuler`', '`rotation.byAxisAngle`').
In order to draw  good IPF maps according to the software used for acquisition, I have to choose the 5th configuration.
![RvRMz3zbQF0KegXV](https://user-images.githubusercontent.com/87763359/148461216-40857628-a255-4645-b548-630f6d5c69a5.png)
Howether, I have to apply some rotations only on axis to replace the rolling direction (RD) of my sample before processing texture analysis.

For instance, if my RD corresponds to the icon's Z, I need to _(i)_ rotate 90° arround X axis to put RD '`south`' and also _(ii)_ rotate 90° arround Y to have the two others directions (ND and TD) on '`east`' and '`outOfPlane`'.
But I can't find solution to do that without rotating IPF maps because of the rotation _(ii)_.

Atex and OIM can rotate only axis angle OR Euler angle, but I didn't get the solution to do it with MTEX.

Could you help me with this point ?
Thanks a lot.

Best regards,
F.

**Comment by kilir** [comment]
Hi,
I'm only halfway sureif I understand all correctly, but there's a difference between plotting conventions (see mtex_setting.m) or whatever you specified in the input wizard and rotating orientations or rotating your EBSD coordinates or both. I think you are mixing things up.

Cheers,
Rüdiger
